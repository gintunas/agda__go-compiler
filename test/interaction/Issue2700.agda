open import Agda.Builtin.Nat

record Pointed (A : Set) : Set where
 field point : A

it : ∀ {A : Set} {{x : A}} → A
it {{x}} = x

instance _ = record { point = 3 - 4 }

_ : Pointed Nat
_ = {!!}
