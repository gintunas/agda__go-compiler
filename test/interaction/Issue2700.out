(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Pointed Nat " nil)
((last . 1) . (agda2-goals-action '(0)))
(agda2-status-action "")
(agda2-info-action "*Goal type etc.*" "Goal: Pointed Nat Elaborates to: record { point = 0 } ————————————————————————————————————————————————————————————" nil)
